doc/tutorials/tutorial_X8.rst

```diff
@@ -1,18 +1,17 @@
 .. _starship:
 
-Executing programs on photonic hardware
-=======================================
+Executing programs on X8 devices
```

Comment:
Not a huge deal, but it still kinda reads a bit weird. This section refers to "X8 devices" and "X8 family", but everywhere else just refers to "X8 chip" etc. (singular).

Suggest moving the "What's in a name" callout to here (with the reduced size per my code suggestion)

Reply:
Have moved the reduced size callout up here :)

Reply:
I've also gone through and standardized any weird appearance of 'X8 chip'